examples/next-openai/app/assistant/page.tsx

```diff
@@ -11,18 +11,22 @@ const roleToColorMap: Record<Message['role'], string> = {
 };
 
 export default function Chat() {
-  const { status, messages, input, submitMessage, handleInputChange, error } =
```

Comment:
I've removed `input` and `handleInputChange` for me the hook should not be responsible of that, what do you think ?